drivers/gpio/gpio_nrf5.c

```diff
@@ -175,6 +182,10 @@ static int gpio_nrf5_config(struct device *dev,
 			pull = GPIO_PULL_DOWN;
 		}
 
+		if (sense == GPIO_SENSE_INVALID) {
+			sense = GPIO_SENSE_DISABLE;
+		}
```

Comment:
An assertion would be better than such silent correction. User should be notified that the passed `flags` value is not valid.